### ISSUE TITLE
[Sourcing][Accessibility][High][Jul] Single selection focus not visible on table

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10001]
#### Is this a bug, enhancement, or feature request?
Bug

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
All these were tested as on 31st May 2023 on https://sap.github.io/fundamental-ngx/#/platform/table/ platform table component samples

#### If this is a bug, please provide steps for reproducing it.
**Issue 5:** On accessing the first column of the selected row, the keyboard focus is not visible to the users.
<img width="452" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/88182358/15a5b565-9376-4dee-936d-b6d053788bb5">

#### Please provide relevant source code if applicable.
The screenshots are attached for references from JAWS screenreader version 2023.

**Comment by fkolar** [comment]
Tested and works for me, tested on SAFARI + CHROME
https://sap.github.io/fundamental-ngx/#/platform/table/row-selection

> When I focus on the table and I can navigate by Arrow Keys and I can see blue rectangle. 

<img width="1159" alt="Screenshot 2023-06-26 at 12 13 46" src="https://github.com/SAP/fundamental-ngx/assets/17149942/62592808-ad04-42ff-95a1-1a5bd85b69be">


@I543348, closing, please feel free to re-open if there is some new evidence or if there is some information that I missed.

**Comment by I543348** [comment]
Tested and works fine for me too.
Thanks Team!

**Comment by sagar7259** [comment]
The focus is now visible cleary on the check boxes, hence this can be closed.

Thanks!
Sagar(ATL team)
